supervisely/convert/base_converter.py

```diff
@@ -141,18 +142,14 @@ def create_empty_annotation(self) -> Annotation:
     def __init__(
         self,
         input_data: str,
-        labeling_interface: Literal[
-            "default",
-            "multi_view",
-            "multispectral",
-            "images_with_16_color",
-            "medical_imaging_single",
-        ] = "default",
+        labeling_interface: Optional[LabelingInterface] = LabelingInterface.DEFAULT,
     ):
         self._input_data: str = input_data
         self._items: List[self.BaseItem] = []
         self._meta: ProjectMeta = None
-        self._labeling_interface: str = labeling_interface
+        if labeling_interface is None:
+            labeling_interface = LabelingInterface.DEFAULT
+        self._labeling_interface = str(labeling_interface)
```

Comment:
get rid of self._labeling_interface = str(...) construction

prefered construction:
```
self._labeling_interface = labeling_interface or LabelingInterface.DEFAULT
```

Reply:
or 
```
self._labeling_interface = labeling_interface or LabelingInterface.DEFAULT.value
```